interfaces/matlab/toolbox/@Reactor/Reactor.m

```diff
@@ -18,33 +18,42 @@
 %     Instance of class :mat:func:`Solution` representing the contents of the
 %     reactor
 % :param typ:
-%     Integer, reactor type. Options are:
+%     String, reactor type. Options are:
 %
-%     1. Reservoir
-%     2. Reactor
-%     3. Flow Reactor
-%     4. Constant Pressure Reactor
-%     5. Ideal Gas Reactor
-%     6. Ideal Gas Constant Pressure Reactor
+%     "Reservoir"
+%     "Reactor"
+%     "FlowReactor"
+%     "ConstPressureReactor"
+%     "IdealGasReactor"
+%     "IdealGasConstPressureReactor"
 %
 % :return:
 %     Instance of class :mat:func:`Reactor`
 %
 
 if nargin == 0
     contents = 0;
-    typ = 2;
+    typ = "Reactor";
```

Comment:
One potential issue here is that "strings" (double-quoted) are only supported in Matlab 2016b or newer -- Matlab 2016a and earlier only have "character arrays (single quoted) which behave a bit differently. I'm not sure what our minimum Matlab version is at present (the [website](https://cantera.org/compiling/dependencies.html#optional-programs) claims it works with 2009a, but I'm not sure if that's still true, and don't think we need to support anything that old in any case), but if we want to use these, we will need to note the version requirement.

Reply:
That's an interesting point. Comparisons of character arrays are a little more annoying than stings. I wasn't aware of the fact that strings were only introduced 3 years ago, but will look into this as I it's not reason enough to lose compatibility.